From aa36b2c0e93a5d2066a7cc4d39a0335f4bc0fe26 Mon Sep 17 00:00:00 2001
From: z00512904 <z00512904@huawei.com>
Date: Wed, 22 Feb 2023 08:02:29 +0000
Subject: [PATCH] [Huawei] Export function sched_setscheduler

Offering:ascend
Reference:NA
Type:bugfix
reason:Export function sched_setscheduler
---
 kernel/sched/core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 4265dc2ab..67387cf86 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -5758,6 +5758,7 @@ int sched_setscheduler(struct task_struct *p, int policy,
 {
 	return _sched_setscheduler(p, policy, param, true);
 }
+EXPORT_SYMBOL_GPL(sched_setscheduler);
 
 int sched_setattr(struct task_struct *p, const struct sched_attr *attr)
 {
-- 
2.17.1

